make

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#Operate in current direcory
+cd $(dirname $0)
+
+# Variables
+LANG="python3"
+MINIFY=0
+DEBUG=0
+
+#Ensure npm installed
+NPM="$(which npm)"
+if [ "$NPM" == "" ]; then
+	echo "npm not installed"
```

Comment:
A tab character (`\t`) doesn't have a standard definition as to how many columns it takes up. So on one machine, it may use two spaces but on another, it would use 4. http://softwareengineering.stackexchange.com/a/66/221365